Lob-API-public.yml

```diff
@@ -321,6 +321,106 @@ tags:
+      | `postcard.in_local_area`          | true      | A postcard receives an "In Local Area" [tracking event](https://docs.lob.com/#tracking_event_object).                                                                                          |
+      | `postcard.processed_for_delivery` | true      | A postcard receives a "Processed for Delivery" [tracking event](https://docs.lob.com/#tracking_event_object)                                                                                   |            
+      | `postcard.re-routed`              | true      | A postcard receives an "Re-Routed" [tracking event](https://docs.lob.com/#tracking_event_object).                                                                                              |
+      | `postcard.returned_to_sender`     | true      | A postcard receives a "Returned to Sender" [tracking event](https://docs.lob.com/#tracking_event_object)                                                                                       |                                                                                                                                  |
+
+
+      **<span style="color:#6683a2">Self Mailers</span>**
+
+      | EVENT TYPE                           | LIVE-ONLY | WHEN EVENT TYPE OCCURS                                                                                                                                                                            |
+      |--------------------------------------|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+      | `self_mailer.created`                | false     | A self mailer is successfully created (Lob returns a 200 status code).                                                                                                                            |
+      | `self_mailer.rendered_pdf`           | false     | A self mailer's PDF proof is successfully rendered.                                                                                                                                               |
+      | `self_mailer.rendered_thumbnails`    | false     | A self mailer's thumbnails are successfully rendered.                                                                                                                                             |
+      | `self_mailer.deleted`                | false     | A self mailer is successfully canceled.                                                                                                                                                           |
+      | `self_mailer.mailed`                 | true      | A self mailer receives a "Mailed" [tracking event](https://docs.lob.com/#tracking_event_object). Only enabled for certain [Print & Mail Editions](https://dashboard.lob.com/#/settings/editions). |
```

Comment:
Noting that this, and others with the keywords "Only enabled for certain", don't have "Only created in the Live Environment" appended.
![image](https://user-images.githubusercontent.com/45194378/127575679-c823bb85-e9e3-4103-99e6-535dc17eaaf5.png)